server/src/cm_job_task.cpp

```diff
@@ -3089,7 +3089,7 @@ tsRenameDB (nvplist * req, nvplist * res, char *_dbmt_error)
     argv[argc++] = newdbname;
     argv[argc++] = NULL;
 
-    snprintf (task_name, TASKNAME_LEN, "%s", "renamedb");
+    strcpy(task_name, "renamedb");
```

Comment:
I think the purpose of the old code was to avoid buffer overrun. However, to achieve that, the size of task_name should be TASKNAME_LEN, not 10. Please update the size.

I propose using strncpy (task_name, const_string, TASKNAME_LEN). Although the copied strings are obviously small enough, this would show a clear intention of avoiding buffer overrun. Maybe an ill-intended developer will want to add a new task with a name longer than 50 character :).